vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```diff
@@ -441,6 +450,9 @@
       CLUSTER_UID: "{{ clusters.resources[0].metadata.uid }}"
       M3MT_NAME: "{{CLUSTER_NAME}}-new-controlplane-image"
       DATA_TEMPLATE_NAME: "{{CLUSTER_NAME}}-controlplane-template"
+      RAW_IMAGE_NAME: "UBUNTU_20.04_NODE_IMAGE_K8S_{{UPGRADED_K8S_VERSION}}-raw.img"
+      IMAGE_URL: "http://172.22.0.1/images/{{ RAW_IMAGE_NAME }}"
+      IMAGE_CHECKSUM: "http://172.22.0.1/images/{{ RAW_IMAGE_NAME }}.{{ IMAGE_CHECKSUM_TYPE }}sum"
```

Comment:
Shouldn't this have the same set of vars as above?

Reply:
Other two variables are not needed in  template file. Those two are necessary during downloading the image.